packages/terra-clinical-item-view/src/ItemView.jsx

```diff
@@ -120,26 +125,65 @@ const defaultEmphasisContentClassesFromIndexes = (rowIndex, rowCount) => {
   return [contentSize, contentColor];
 };
 
-const startEmphasisContentClassesFromIndexes = (rowIndex, rowCount, columnIndex) => {
-  if (columnIndex > 0 || rowIndex >= 2) {
+const startEmphasisContentClassesFromIndexes = (rowIndex, rowCount, contentIndex) => {
+  if (contentIndex > 0 || rowIndex >= 2) {
     return ['content-secondary-size', 'content-secondary-color'];
   }
 
   return defaultEmphasisContentClassesFromIndexes(rowIndex, rowCount);
 };
 
-const classesForContent = (rowIndex, rowCount, columnIndex, emphasis) => {
+const classesForContent = (rowIndex, rowCount, contentIndex, emphasis) => {
   let classes;
 
   if (emphasis === TextEmphasisTypes.START) {
-    classes = startEmphasisContentClassesFromIndexes(rowIndex, rowCount, columnIndex);
+    classes = startEmphasisContentClassesFromIndexes(rowIndex, rowCount, contentIndex);
   } else {
     classes = defaultEmphasisContentClassesFromIndexes(rowIndex, rowCount);
   }
 
   return ['content'].concat(classes);
 };
 
+const twoColumnGrouping = (displays) => {
+  let count = 0;
+  const displayGroups = [];
+  const primaryColumn = [];
+  const secondaryColumn = [];
+
+  while (displays.length) {
+    count += 1;
+
+    if (count % 2 === 0) {
+      secondaryColumn.push(displays.splice(0, 1));
+    } else {
+      primaryColumn.push(displays.splice(0, 1));
+    }
+  }
+
+  displayGroups.push(primaryColumn);
+  displayGroups.push(secondaryColumn);
+
+  return displayGroups;
+};
+
+const renderRow = (row, rowIndex, rowCount, emphasis) => {
+  const rowKey = rowIndex;
+  return (
+    <div className={cx('row')} key={rowKey}>
+      {row.map((display, displayIndex) => {
+        const displayKey = displayIndex;
+        const contentClasses = classesForContent(rowIndex, rowCount, displayIndex, emphasis);
+        return (
```

Comment:
nit: Can we add a new line before `return` for readability?

Reply:
[14c6495](https://github.com/cerner/terra-clinical/pull/912/commits/14c6495877fe6eba310beae0211922b7cfbcec50)